AppInspector.CLI/CLICmdOptions.cs

```diff
@@ -22,26 +22,48 @@ public class CLICommandOptions : CommandOptions
     /// <summary>
     /// CLI command distinct arguments
     /// </summary>
-    [Verb("analyze", HelpText = "Inspect source directory/file/compressed file (.tgz|zip) against defined characteristics")]
-    public class CLIAnalyzeCmdOptions : CLICommandOptions
+    [Verb("get-tags", HelpText = "Inspect source directory/file/compressed file (.tgz|zip) against defined characteristics for unique tags")]
```

Comment:
are there any other tools other than codeinspector using application inspector for automation? If yes, this change will break it when they upgrade, if not nevermind.

Reply:
Why would this break them? The analyze command still exists.

Reply:
To clarify there are some changes to the flags that might break them (only removal no flags have changed meaning).  I don't think there is any actual missing functionality net for net though.  You can use the GetTags command instead of the old AnalyzeCommand with the arguments to limit down to single matches.